docs/DeveloperGuide.md

```diff
@@ -312,49 +307,69 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case ends.
 
-* 3a. The given index is invalid.
+**Use case: UC02 - Delete a profile**
+
+**MSS**
+
+1. User requests to <u>view all profiles (UC01)</u>.
+2. User requests to delete a specific profile in the list.
+3. NUScheduler deletes the person.
 
-    * 3a1. NUScheduler shows an error message.
+    Use case ends.
 
-      Use case resumes at step 2.
+**Extensions**
+
+* 2a. The given index is invalid.
+
+    * 2a1. NUScheduler shows an error message.
+
+      Use case ends.
 
-**Use case: Add an event**
+**Use case: UC03 - Add an event**
 
 **MSS**
 
-1. User requests to add an event to the schedule
-2. NUScheduler adds the event and displays the updated schedule of events
+1. User requests to add an event to the schedule.
+2. NUScheduler adds the event and displays the updated schedule of events.
 
    Use case ends.
 
 **Extensions**
 
-* 1a. The starting date or the ending date is not provided.
+* 1a. The Title, starting date or the ending date is not provided.
 
     * 1a1. NUScheduler reminds user of the correct format to add an event.
 
       Use case ends.
 
-**User case: View upcoming events**
+* 1b. An invalid profile is provided.
+
+    * 1b1. NUScheduler shows an error message.
+
+      Use case ends.
+
+**User case: UC04 - View upcoming events**
 
 **MSS**
 
-1. User requests to view all upcoming events within the next specified days
-2. NUScheduler displays the filtered list of matching events
+1. User requests to view all upcoming events within the next specified days.
+2. NUScheduler displays the filtered list of matching events.
 
 **Extensions**
 
-* 1a. The schedule is empty.
+* 1a. The given time period is invalid.
 
-  Use case ends.
+    * 1a1. NUScheduler shows an error message.
 
-* 1b. The given time period is invalid.
+      Use case ends.
+    
+* 2a. The schedule is empty.
 
-    * 1b1. NUScheduler shows an error message.
+  Use case ends.
 
-      Use case ends.
+* 2b. There are no events in the next specified days.
```

Comment:
Is similar to `2a`?

Reply:
Yea I think I agree its quite similar but I interpreted 2a as meaning the entire schedule being empty so I added another extension where nothing is shown. Maybe I could combine both into `There are no events in the next specified days` which covers both situation?

Reply:
I think that's a good suggestion, and after your comment, I think the current more verbose version is fine as well. What do the rest think?